docs/specs/toc.yml

```diff
@@ -1399,3 +1399,30 @@ outputs:
     {"_tocRel": "f2/toc.json"}
   docs/f1/toc.json:
   docs/f2/toc.json:
+---
+# turn skip-level toc error to warning
+inputs:
+  docfx.yml: |
+    rules:
+      invalid-toc-level: warning
+  docs/TOC.md: |
+    # level 1
```

Comment:
can the first level be `##` ?

Reply:
yes, check test `more than 6 leading count of #`

Reply:
Is it implemented like that or designed like that, @superyyrrzz what does v2 do here?

Reply:
tested here: https://github.com/MicrosoftDocs/engineering/pull/514#issuecomment-505732272

Reply:
supported in v2, built page: https://review.docs.microsoft.com/en-us/engineering/projects/toc.json?branch=pr-en-us-514